noxfile.py

```diff
@@ -30,24 +29,21 @@ def _expand_env_vars(file_path: Path):
 
 
 @nox.session(python=["3.8", "3.9", "3.10"])
-@nox.parametrize("airflow", ["2.2.4", "2.3", "2.4", "2.5"])
+@nox.parametrize("airflow", ["2.3", "2.4", "2.5", "2.6", "2.7", "2.8"])
```

Comment:
We can remove 2.5 and 2.6 since they aren't LTS: https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule

Can happen in a separate PR

Reply:
Created ticket to track this: https://github.com/astronomer/astro-provider-databricks/issues/67